core/backend/src/BackendHubAccess.ts

```diff
@@ -197,11 +197,6 @@ export interface BackendHubAccess {
    * @internal
    */
   queryV2Checkpoint: (arg: CheckpointProps) => Promise<V2CheckpointAccessProps | undefined>;
-  /**
-   * download a v2 checkpoint
-   * @internal
-   */
-  downloadV2Checkpoint: (arg: CheckpointArg) => Promise<ChangesetIndexAndId>;
```

Comment:
I think this is going to require a change in the client libs for the iModels API.  FYI @austeja-bentley @wgoehrig